examples/units/evans_test.py

```diff
@@ -1,8 +1,13 @@
 """
+==========
+Evans test
```

Comment:
Who or what is "Evans"? Does anyone know? @tacaswell? I think this needs a different name.